helper/parseItems.js

```diff
@@ -1,6 +1,6 @@
 const { decodeItems, decodeItem } = require('./decode');
 
-const parseItems = async (profileData, museumData, options = { removeEmptyItems: true, combineStorage: true, additionalInventories: null }) => {
+const parseItems = async (profileData, museumData, options = { removeEmptyItems: true, combineStorage: true, additionalInventories: null, parsedInventories: null }) => {
```

Comment:
Um, on second thought and further review, this is probably useless here. I will make a comment somewhere else. This can probably be removed.